### PR TITLE
application-prod.properties corrected

### PR DESCRIPTION
**Pull request by yapollo2002**

### CODE REVIEW

**Review by Farleo — APPROVED** [review]

File: .github/workflows/aws.yml [hunk 1]
```diff
@@ -79,6 +79,9 @@ jobs:
...
+      
+    - name: Finish Dockerfile
+      run: echo ENTRYPOINT ["java","-Dspring.security.oauth2.client.registration.google.clientId=${{ secrets.GOOGLE_CLIENT_ID }}", "-Dspring.security.oauth2.client.registration.google.clientSecret=${{ secrets.GOOGLE_CLIENT_SECRET }}", "-DAWS_ACCESS_KEY_ID=${{ secrets.AWS_ACCESS_KEY_ID }}", "-DAWS_SECRET_ACCESS_KEY=${{ secrets.AWS_SECRET_ACCESS_KEY }}", "-Dspring.security.oauth2.client.registration.facebook.clientId=${{ secrets.FACEBOOK_CLIENT_ID }}", "-Dspring.security.oauth2.client.registration.facebook.clientSecret=${{ secrets.FACEBOOK_CLIENT_SECRET }}", "-Dspring.security.oauth2.client.registration.github.clientId=${{ secrets.GIT_HUB_CLIENT_ID }}", "-Dspring.security.oauth2.client.registration.github.clientSecret=${{ secrets.GIT_HUB_CLIENT_SECRET }}", "-Daws.users-folder=user-profile-picture", "-Daws.bucket-name=course-evaluation-storage", "-Dspring.mail.username=noreply.courseevaluation@gmail.com", "-Dspring.mail.password=java128if", "-DAWS_RDS_USERNAME=${{secrets.AWS_RDS_PASSWORD}}", "-DAWS_RDS_PASSWORD=${{secrets.AWS_RDS_PASSWORD}}", -Dspring.profiles.active=prod"-jar","course_evaluation.jar"] >> Dockerfile
```
Comment (Farleo) [inline .github/workflows/aws.yml:84]: try hide this => -Dspring.mail.password=

File: src/main/resources/application-prod.properties [hunk 2]
```diff
@@ -54,11 +57,11 @@ server.tomcat.max-http-form-post-size=10MB
 max-image-size-in-mb=5
 
 # AWS credentials
-aws.accessKey=${AWS_ACCESS_KEY}
-aws.secretKey=${AWS_SECRET_KEY}
+aws.accessKey=${AWS_ACCESS_KEY_ID}
```
Comment (yapollo2002) [inline src/main/resources/application-prod.properties:60]: ok

File: src/main/resources/application-prod.properties [hunk 3]
```diff
@@ -54,11 +57,11 @@ server.tomcat.max-http-form-post-size=10MB
...
+aws.secretKey=${AWS_SECRET_ACCESS_KEY}
 
 # AWS folder name for storing users profile picture
-aws.users-folder=${AWS_USERS_FOLDER}
+aws.users-folder=user-profile-picture
```
Comment (yapollo2002) [inline src/main/resources/application-prod.properties:64]: ok

File: src/main/resources/application-prod.properties [hunk 4]
```diff
@@ -18,10 +19,12 @@ spring.security.oauth2.client.registration.github.clientSecret=${GIT_HUB_CLIENT_
 spring.security.oauth2.client.registration.github.scope.=user:email, read:user
 
 spring.liquibase.change-log=classpath:db/changelog/db.changelog-master.xml
-spring.liquibase.url=jdbc:postgresql://evaluationsystem.ckewj8xlh5oj.eu-central-1.rds.amazonaws.com:5432/postgres
+spring.liquibase.url=jdbc:postgresql://evaluationsystem.ckewj8xlh5oj.eu-central-1.rds.amazonaws.com:5432/course_evaluation
```
Comment (yapollo2002) [inline src/main/resources/application-prod.properties:22]: ok

File: src/main/resources/application-prod.properties [hunk 5]
```diff
@@ -1,8 +1,9 @@
 spring.application.name=course_evaluation
-spring.datasource.url=jdbc:postgresql://evaluationsystem.ckewj8xlh5oj.eu-central-1.rds.amazonaws.com:5432/postgres
+spring.datasource.url=jdbc:postgresql://evaluationsystem.ckewj8xlh5oj.eu-central-1.rds.amazonaws.com:5432/course_evaluation
```
Comment (yapollo2002) [inline src/main/resources/application-prod.properties:2]: ok

File: src/main/resources/application-prod.properties [hunk 6]
```diff
@@ -1,8 +1,9 @@
...
 spring.datasource.username=${AWS_RDS_USERNAME}
 spring.datasource.password=${AWS_RDS_PASSWORD}
 
+
```
Comment (yapollo2002) [inline src/main/resources/application-prod.properties:6]: ok

File: src/main/resources/application-prod.properties [hunk 7]
```diff
@@ -54,11 +57,11 @@ server.tomcat.max-http-form-post-size=10MB
 max-image-size-in-mb=5
 
 # AWS credentials
-aws.accessKey=${AWS_ACCESS_KEY}
-aws.secretKey=${AWS_SECRET_KEY}
+aws.accessKey=${AWS_ACCESS_KEY_ID}
+aws.secretKey=${AWS_SECRET_ACCESS_KEY}
```
Comment (yapollo2002) [inline src/main/resources/application-prod.properties:61]: ok

File: .github/workflows/aws.yml (hunk 1 above)
Comment (yapollo2002) [reply .github/workflows/aws.yml:84]: ok

**Pull request merged**